pom.xml

```diff
@@ -144,6 +144,7 @@ flexible messaging model and an intuitive client API.</description>
     <hbc-core.version>2.2.0</hbc-core.version>
     <cassandra-driver-core.version>3.4.0</cassandra-driver-core.version>
     <aerospike-client.version>4.1.5</aerospike-client.version>
+    <kafka-client.version>0.10.0.0</kafka-client.version>
```

Comment:
shall we just use 1.0?

Reply:
1.0 seems a very recent release. Used 0.10 since that seems to be the most popular version out there. I'm assuming that most of the migrations will come from ppl running kafka for a while.

Reply:
There's already a version of kafka clients used for the wrapper. We should probably standardize on one version